docs/user-guide/css-processors.md

```diff
@@ -17,10 +17,10 @@ By default, the linter can *parse* any the following non-standard syntaxes by us
 -   SCSS (using [`postcss-scss`](https://github.com/postcss/postcss-scss))
 -   Less (using [`postcss-less`](https://github.com/shellscape/postcss-less))
 -   SugarSS (using [`sugarss`](https://github.com/postcss/sugarss))
--   HTML,
```

Comment:
Let's remove "Javascript or typescript" from this list for now as it's an undocumented feature. And let's remove "Markdown" and "HTML-like" as we're still resolving the processor vs syntax issue.